nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/cli/StopDomainCommand.java

```diff
@@ -167,17 +167,17 @@ protected int dasNotRunning() throws CommandException {
     protected void doCommand() throws CommandException {
         // run the remote stop-domain command and throw away the output
         RemoteCLICommand cmd = new RemoteCLICommand(getName(), programOpts, env);
+        File watchedPid = isLocal() ? getServerDirs().getPidFile() : null;
```

Comment:
First look - if this becomes `null`, won't `ProcessUtils.isAlive(watchedPid)` throw `NPEx` on `pidFile.exists()`?